### PR TITLE
pam/qrcodemodel: Use normal-size Qr code when in simple terminals

### PR DESCRIPTION
**Pull request by 3v1n0**
If the terminal is not advanced enough to support the more complex utf-8
characters needed to print the small qrcode, just use the simpler
implementation via the big qrcode that uses characters that seems to be
supported everywhere.

In fact bigger qrcode use two times the character `█` to do a square, and
this is supported by the normal tty fonts:

    getunimap | grep █ 0x0db	U+2588	# █
    setfont -v -ou /dev/stdout | grep 0x0db

Without having to go through building one via `#`'s.

In theory we can inspect for font support on that, but it's not something we want to
implement for now.

UDENG-3109

Closes: #361

### CODE REVIEW

**Review by denisonbarbosa — APPROVED** [review]
Looks good to me. Thanks for fixing this!

**Review by didrocks — COMMENTED** [review]
Nice work! I do have one question before approving!

File: pam/internal/adapter/qrcodemodel.go [hunk 1]
```diff
@@ -81,14 +83,29 @@ func (m qrcodeModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	return m, cmd
 }
 
+func (m qrcodeModel) renderQrCode() (qr string) {
+	defer func() { qr = strings.TrimRight(qr, "\n") }()
+
+	if os.Getenv("XDG_SESSION_TYPE") == "tty" {
```
[the remark this answers is not in the document]
Comment (3v1n0) [reply pam/internal/adapter/qrcodemodel.go:89]: > case wwhere we have that env variable exported and in another more modern terminal env?

One case I saw this not to be true is xterm or screen, so in such cases I feel it's always better to fallback to the safer qrcode version.

[the remark this answers is not in the document]
Comment (didrocks) [reply pam/internal/adapter/qrcodemodel.go:89]: Interesting! Mind adding a comment to it?

[the remark this answers is not in the document]
Comment (3v1n0) [reply pam/internal/adapter/qrcodemodel.go:89]: sure i was on it.

[the remark this answers is not in the document]
Comment (3v1n0) [reply pam/internal/adapter/qrcodemodel.go:89]: done

**Review by didrocks — APPROVED** [review]
Thanks for the comment! Nice work

**Pull request merged**